src/domain/event/CreateEventPage.tsx

```diff
@@ -119,6 +122,7 @@ const CreateEventPage: React.FC = () => {
     } catch (e) {
       // TODO: Improve error handling when API returns more informative errors
       if (process.env.NODE_ENV === 'test') {
```

Comment:
Maybe helper function to test is it testing environment

Reply:
fixed